### PR TITLE
magento/devdocs#: Locate templates, layouts, and styles

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information how to enable/disable templates hints via CLI.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.1/frontend-dev-guide/themes/debug-theme.html
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/themes/debug-theme.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/debug-theme.html

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/frontend-dev-guide/themes/debug-theme.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ To enable template hints:
 
 ![Enabling template hints]({{ site.baseurl }}/common/images/fdg_debug_theme.png)
 
+{:.bs-callout .bs-callout-info}
+You can also use CLI to enable/disable template hints.
+
+Use `bin/magento dev:template-hints:enable` command in CLI to enable template hints, and `bin/magento dev:template-hints:disable` to disable them. Do not forget to clear cache after command run. For example: 
+
+```bash
+bin/magento dev:template-hints:enable
```
Comment (ronak2ram) [inline guides/v2.1/frontend-dev-guide/themes/debug-theme.md]: Hi @atwixfirster 
This command does not work with Magento v2.1. so can you please update this change for doc v2.2 and v2.3.

Comment (atwixfirster) [reply guides/v2.1/frontend-dev-guide/themes/debug-theme.md]: > This command does not work with Magento v2.1. 

yay!!!

> so can you please update this change for doc v2.2 and v2.3.

**done**

Please review. 

Thank you. @ronak2ram

Comment (ronak2ram) [reply guides/v2.1/frontend-dev-guide/themes/debug-theme.md]: thanks for the quick update @atwixfirster

**Comment by atwixfirster** [comment]
**done**

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**